pxf/pxf-service/src/main/resources/pxf-profiles-default.xml

```diff
@@ -177,11 +177,11 @@ under the License.
     </profile>
     <profile>
         <name>Jdbc</name>
-        <description>A profile for reading data into HAWQ via JDBC</description>
+        <description>A profile to access (read & write) data via JDBC</description>
```

Comment:
the "&" character here breaks XML and the PXF server does not even startup, please remove it.

Reply:
or may be change `&` to `&amp;` ?

Reply:
this is supposed to be user readable, rather than machine readable, just substitute with word "and" to avoid issues.